tests/test_cli_get_set.py

```diff
@@ -300,9 +432,9 @@ def test_set_override_existing_file(self):
         self.assertFalse(os.path.isfile(self.path))
         self.assertTrue(os.path.isfile(override))
         with open(override, 'r') as f:
-            out = f.read()
-            self.assertIn('network:\n  ethernets:\n    eth0:\n      dhcp4: false', out)  # new
-            self.assertIn('eth1:\n      dhcp6: false', out)  # old
+            out = yaml.safe_load(f)
+            self.assertIs(False, out['network']['ethernets']['eth0']['dhcp4'])
+            self.assertIs(False, out['network']['ethernets']['eth1']['dhcp6'])
```

Comment:
(non-blocking): I think we could be using `assertFalse()` here and below.

Reply:
Sadly, no. `assertFalse` and `assertTrue` coerce the tested value into a boolean, and we want to make sure we have the actual boolean values here, not any truthy or falsy value.